pkg/constant/undo.go

```diff
@@ -21,6 +21,37 @@ const (
 	DeleteFrom                     = "DELETE FROM "
 	DefaultTransactionUndoLogTable = " undo_log "
```

Comment:
sql expression can't put it into constant pkg

Reply:
pls put sql with data model

Reply:
ok, thx

Reply:
done